lldb/source/Plugins/Process/elf-core/ProcessElfCore.cpp

```diff
@@ -224,7 +224,7 @@ Status ProcessElfCore::DoLoadCore() {
   ArchSpec core_arch(m_core_module_sp->GetArchitecture());
   target_arch.MergeFrom(core_arch);
   GetTarget().SetArchitecture(target_arch);
- 
+
```

Comment:
Keep formatting changes separate, push them directly if you have the rights to do so. If you have to choose between a clean commit and the formatter complaining, let it complain :)

Reply:
This pull request will be rebased onto #117028 once it landed onto main, which will do clang-format.